src/main/kotlin/gg/essential/elementa/components/ScrollComponent.kt

```diff
@@ -20,18 +22,64 @@ import kotlin.math.max
  *
  * Also prevents scrolling past what should be reasonable.
  */
-class ScrollComponent @JvmOverloads constructor(
+class ScrollComponent constructor(
     emptyString: String = "",
     private val innerPadding: Float = 0f,
     private val scrollIconColor: Color = Color.WHITE,
-    private val horizontalScrollEnabled: Boolean = false,
-    private val verticalScrollEnabled: Boolean = true,
+    private val scrollDirection: Direction = Direction.PreferVertical,
```

Comment:
Why not keep the `Vertical` default? Seems less surprising than vertical-but-it-actually-also-works-horizontally-even-though-you-didn't-ask-for-that.